CHANGELOG.md

```diff
@@ -38,25 +38,25 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - uuid: possible unsoundness.
 - query: avoid panics during `Query::bind()` calls ([#103]).
 
-[#103]: https://github.com/loyd/clickhouse.rs/issues/103
-[#102]: https://github.com/loyd/clickhouse.rs/pull/102
-[#91]: https://github.com/loyd/clickhouse.rs/pull/91
-[#90]: https://github.com/loyd/clickhouse.rs/pull/90
-[#89]: https://github.com/loyd/clickhouse.rs/issues/89
-[#83]: https://github.com/loyd/clickhouse.rs/pull/83
+[#103]: https://github.com/ClickHouse/clickhouse.rs/issues/103
```

Comment:
@slvrtrn should we rename the repository to `clickhouse-rs` to keep it consistent with `clickhouse-go` and `clickhouse-js`?

Reply:
It would create confusion with https://github.com/suharev7/clickhouse-rs (especially on crates.io), wouldn't it? If so, I'm down for keeping the current name, even though it's not aligned with other projects

Reply:
> especially on crates.io

The crate's name on crates.io is unrelated to the repository name. This crate is published under the `clickhouse` crate name.

Reply:
Anyway, both of you think it's better to leave the current name, so ok, but it makes naming inside `ClickHouse/` org inconsistent =(

Reply:
> The crate's name on crates.io is unrelated to the repository name. 

if so, let's rename